01-report.js

```diff
@@ -11,15 +11,17 @@ export function getLast12MonthReport(database) {
     // Please don't get stuck too long on this one, ask for help if you need it.
     // Also, reading the doc might be helpful :D
     const today = new Date();
-    const d = new Date();
-    const year = d.getFullYear();
+    let date = new Date(); 
+    const year = date.getFullYear();
+    let month = date.getMonth();
```

Comment:
Why did you declare the month here ? What's the difference with *inside* the while.